### ISSUE TITLE
Click on database name does not work, error: 200, text: OK - only PMA 4.6.0rc1

### ISSUE THREAD

**Issue by 2002demon** [phpmyadmin/phpmyadmin#12057]
### Steps to reproduce
1. Login
2. Click dbname on left side
3. We have error 200, Text:OK

That is new bug in rc1 , 4.6.0alpha is OK.

Console tell us: **Invalid JSON!Unexpected token <**
on get_scripts.js.php?scripts%5B%5D=indexes.js&scripts%5B%5D=common.js&scripts%5B%5D=page_settings.js&scripts%5B%5D=codemirror/lib/codemirror.js&scripts%5B%5D=codemirror/mode/sql/sql.js&scripts%5B%5D=codemirror/addon/runmode/runmode.js&scripts%5B%5D=codemirror/addon/hint/show-hint.js&scripts%5B%5D=codemirror/addon/hint/sql-hint.js&scripts%5B%5D=codemirror/addon/lint/lint.js&scripts%5B%5D=codemirror/addon/lint/sql-lint.js&scripts%5B%5D=console.js&v=4.6.0-rc1

Ajax return **HTML with json** on
db_structure.php?server=1&db=panel&token=3f1b9ae121d777fb34f410c3ee1eaca&ajax_request=true&ajax_page_request=true&_nocache=1456836173294737625
```
<form method="post" action="db_structure.php" name="tablesForm" id="tablesForm">
<input type="hidden" name="db" value="panel" /><input type="hidden" name="token" value="3f1b9ae121d777f7b34f410c3ee1eaca" /><table class="data">
    <thead>
        <tr>
            <th class="print_ignore"></th>
            <th>{"message":"<div id=\"page_settings_modal\"><div class=\"page_settings\"><form method=\"post\" action=\"db_structure.php?db=panel&amp;table=&amp;server=1&amp;target=&amp;token=3f1b9ae121d777f7b34f410c3ee1eaca\" class=\"config-form disableAjax\"><input type=\"hidden\" name=\"tab_hash\" value=\"\" \/><input type=\"hidden\" name=\"check_page_refresh\"  id=\"check_page_refresh\" value=\"\" \/>\n<input type=\"hidden\" name=\"token\" value=\"3f1b9ae121d777f7b34f410c3ee1eaca\" \/>\n<input type=\"hidden\" name=\"submit_save\" value=\"DbStructure\" \/><ul class=\"tabs\"    ><li     ><a href=\"#DbStructure\"                                >Datenbank-Struktur<\/a><\/li><\/ul><br clear=\"right\" \/><div class=\"tabs_contents\"><fieldset class=\"optbox\" id=\"DbStructure\"><legend>Datenbank-Struktur<\/legend><p>W\u00e4hlen Sie aus, welche Details in der Datenbankstrukturansicht (Liste der Tabellen) angezeigt werden sollen.<\/p><table width=\"100%\" cellspacing=\"0\"><tr><th><label for=\"ShowDbStructureComment\">Tabellen-Kommentare einblenden<\/label><span class=\"doc\"><a href=\".\/doc\/html\/config.html#cfg_ShowDbStructureComment\" target=\"documentation\"><img src=\"themes\/dot.gif\" title=\"Dokumentation\" alt=\"Dokumentation\" class=\"icon ic_b_help\" \/><\/a>\n<\/span><small>Eine Spalte mit den Kommentaren f\u00fcr alle Tabellen ein-\/ausblenden.<\/small><\/th><td><span class=\"checkbox\"><input type=\"checkbox\" name=\"ShowDbStructureComment\" id=\"ShowDbStructureComment\" \/><\/span><a class=\"restore-default\" href=\"#ShowDbStructureComment\" title=\"Voreingestellten Wert wiederherstellen\" style=\"display:none\"><img src=\"themes\/dot.gif\" title=\"\" alt=\"\" class=\"icon ic_s_reload\" \/><\/a><\/td><\/tr><tr><th><label for=\"ShowDbStructureCreation\">Erstellungs-Zeitstempel anzeigen<\/label><span class=\"doc\"><a href=\".\/doc\/html\/config.html#cfg_ShowDbStructureCreation\" target=\"documentation\"><img src=\"themes\/dot.gif\" title=\"Dokumentation\" alt=\"Dokumentation\" class=\"icon ic_b_help\" \/><\/a>\n<\/span><small>Eine Spalte mit dem Erstellungs-Zeitstempel f\u00fcr alle Tabellen ein-\/ausblenden.<\/small><\/th><td><span class=\"checkbox\"><input type=\"checkbox\" name=\"ShowDbStructureCreation\" id=\"ShowDbStructureCreation\" \/><\/span><a class=\"restore-default\" href=\"#ShowDbStructureCreation\" title=\"Voreingestellten Wert wiederherstellen\" style=\"display:none\"><img src=\"themes\/dot.gif\" title=\"\" alt=\"\" class=\"icon ic_s_reload\" \/><\/a><\/td><\/tr><tr><th><label for=\"ShowDbStructureLastUpdate\">Zeitstempel des letzten Updates anzeigen<\/label><span class=\"doc\"><a href=\".\/doc\/html\/config.html#cfg_ShowDbStructureLastUpdate\" target=\"documentation\"><img src=\"themes\/dot.gif\" title=\"Dokumentation\" alt=\"Dokumentation\" class=\"icon ic_b_help\" \/><\/a>\n<\/span><small>Eine Zeile mit dem Zeitstempel des letzten Updates f\u00fcr alle Tabellen ein-\/ausblenden.<\/small><\/th><td><span class=\"checkbox\"><input type=\"checkbox\" name=\"ShowDbStructureLastUpdate\" id=\"ShowDbStructureLastUpdate\" \/><\/span><a class=\"restore-default\" href=\"#ShowDbStructureLastUpdate\" title=\"Voreingestellten Wert wiederherstellen\" style=\"display:none\"><img src=\"themes\/dot.gif\" title=\"\" alt=\"\" class=\"icon ic_s_reload\" \/><\/a><\/td><\/tr><tr><th><label for=\"ShowDbStructureLastCheck\">Zeitstempel der letzten \u00dcberpr\u00fcfung einblenden<\/label><span class=\"doc\"><a href=\".\/doc\/html\/config.html#cfg_ShowDbStructureLastCheck\" target=\"documentation\"><img src=\"themes\/dot.gif\" title=\"Dokumentation\" alt=\"Dokumentation\" class=\"icon ic_b_help\" \/><\/a>\n<\/span><small>Eine Zeile mit dem Zeitstempel der letzten \u00dcberpr\u00fcfung f\u00fcr alle Tabellen ein-\/ausblenden.<\/small><\/th><td><span class=\"checkbox\"><input type=\"checkbox\" name=\"ShowDbStructureLastCheck\" id=\"ShowDbStructureLastCheck\" \/><\/span><a class=\"restore-default\" href=\"#ShowDbStructureLastCheck\" title=\"Voreingestellten Wert wiederherstellen\" style=\"display:none\"><img src=\"themes\/dot.gif\" title=\"\" alt=\"\" class=\"icon ic_s_reload\" \/><\/a><\/td><\/tr><\/table><\/fieldset><\/div>\n<\/form>\n<script type=\"text\/javascript\">if (typeof configInlineParams === \"undefined\" || !Array.isArray(configInlineParams)) configInlineParams = [];\nconfigInlineParams.push(function() {\n$.extend(PMA_messages, {\n\t'error_nan_p': 'Keine positive Zahl!',\n\t'error_nan_nneg': 'Keine nicht-negative Zahl!',\n\t'error_incorrect_port': 'Keine g\u00fcltige Portnummer!',\n\t'error_invalid_value': 'Ung\u00fcltiger Wert!',\n\t'error_value_lte': 'Wert muss gleich oder kleiner als %s sein!'});\n$.extend(defaultValues, {\n\t'ShowDbStructureComment': false,\n\t'ShowDbStructureCreation': false,\n\t'ShowDbStructureLastUpdate': false,\n\t'ShowDbStructureLastCheck': false});\n});\nif (typeof configScriptLoaded !== \"undefined\" && configInlineParams) loadInlineConfig();<\/script><\/div><\/div><div id=\"tableslistcontainer\">","success":true,"_title":"<title> \/ localhost \/ panel | phpMyAdmin 4.6.0-rc1<\/title>","_menuHash":"6020f93b","_menu":"<div id='floating_menubar'><\/div><div id='serverinfo'><img src=\"themes\/dot.gif\" title=\"\" alt=\"\" class=\"icon ic_s_host item\" \/><a href=\"index.php?token=3f1b9ae121d777f7b34f410c3ee1eaca\" class=\"item\">Server: localhost<\/a><span class='separator item'>&nbsp;\u00bb<\/span><img src=\"themes\/dot.gif\" title=\"\" alt=\"\" class=\"icon ic_s_db item\" \/><a href=\"db_structure.php?db=panel&amp;token=3f1b9ae121d777f7b34f410c3ee1eaca\" class=\"item\">Datenbank: panel<\/a><div class=\"clearfloat\"><\/div><\/div><div id=\"topmenucontainer\" class=\"menucontainer\"><ul id=\"topmenu\"  class=\"resizable-menu\"><li      class=\"active\"><a href=\"db_structure.php?db=panel&amp;token=3f1b9ae121d777f7b34f410c3ee1eaca\"                         class=\"tabactive\"        ><img src=\"themes\/dot.gif\" title=\"Struktur\" alt=\"Struktur\" class=\"icon ic_b_props\" \/>&nbsp;Struktur<\/a><\/li><li     ><a href=\"db_sql.php?db=panel&amp;token=3f1b9ae121d777f7b34f410c3ee1eaca\"                         class=\"tab\"        ><img src=\"themes\/dot.gif\" title=\"SQL\" alt=\"SQL\" class=\"icon ic_b_sql\" \/>&nbsp;SQL<\/a><\/li><li     ><a href=\"db_search.php?db=panel&amp;token=3f1b9ae121d777f7b34f410c3ee1eaca\"                         class=\"tab\"        ><img src=\"themes\/dot.gif\" title=\"Suche\" alt=\"Suche\" class=\"icon ic_b_search\" \/>&nbsp;Suche<\/a><\/li><li     ><a href=\"db_qbe.php?db=panel&amp;token=3f1b9ae121d777f7b34f410c3ee1eaca\"                         class=\"tab\"        ><img src=\"themes\/dot.gif\" title=\"Abfrage\" alt=\"Abfrage\" class=\"icon ic_s_db\" \/>&nbsp;Abfrage<\/a><\/li><li     ><a href=\"db_export.php?db=panel&amp;token=3f1b9ae121d777f7b34f410c3ee1eaca\"                         class=\"tab\"        ><img src=\"themes\/dot.gif\" title=\"Exportieren\" alt=\"Exportieren\" class=\"icon ic_b_export\" \/>&nbsp;Exportieren<\/a><\/li><li     ><a href=\"db_import.php?db=panel&amp;token=3f1b9ae121d777f7b34f410c3ee1eaca\"                         class=\"tab\"        ><img src=\"themes\/dot.gif\" title=\"Importieren\" alt=\"Importieren\" class=\"icon ic_b_import\" \/>&nbsp;Importieren<\/a><\/li><li     ><a href=\"db_operations.php?db=panel&amp;token=3f1b9ae121d777f7b34f410c3ee1eaca\"                         class=\"tab\"        ><img src=\"themes\/dot.gif\" title=\"Operationen\" alt=\"Operationen\" class=\"icon ic_b_tblops\" \/>&nbsp;Operationen<\/a><\/li><li     ><a href=\"server_privileges.php?db=panel&amp;token=3f1b9ae121d777f7b34f410c3ee1eaca&amp;checkprivsdb=panel&amp;viewing_mode=db\"                         class=\"tab\"        ><img src=\"themes\/dot.gif\" title=\"Rechte\" alt=\"Rechte\" class=\"icon ic_s_rights\" \/>&nbsp;Rechte<\/a><\/li><li     ><a href=\"db_routines.php?db=panel&amp;token=3f1b9ae121d777f7b34f410c3ee1eaca\"                         class=\"tab\"        ><img src=\"themes\/dot.gif\" title=\"Routinen\" alt=\"Routinen\" class=\"icon ic_b_routines\" \/>&nbsp;Routinen<\/a><\/li><li     ><a href=\"db_events.php?db=panel&amp;token=3f1b9ae121d777f7b34f410c3ee1eaca\"                         class=\"tab\"        ><img src=\"themes\/dot.gif\" title=\"Ereignisse\" alt=\"Ereignisse\" class=\"icon ic_b_events\" \/>&nbsp;Ereignisse<\/a><\/li><li     ><a href=\"db_triggers.php?db=panel&amp;token=3f1b9ae121d777f7b34f410c3ee1eaca\"                         class=\"tab\"        ><img src=\"themes\/dot.gif\" title=\"Trigger\" alt=\"Trigger\" class=\"icon ic_b_triggers\" \/>&nbsp;Trigger<\/a><\/li><li     ><a href=\"db_designer.php?db=panel&amp;token=3f1b9ae121d777f7b34f410c3ee1eaca\"                 id=\"designer_tab\"         class=\"tab\"        ><img src=\"themes\/dot.gif\" title=\"Designer\" alt=\"Designer\" class=\"icon ic_b_relations\" \/>&nbsp;Designer<\/a><\/li><div class=\"clearfloat\"><\/div><\/ul>\n<\/div>\n","_scripts":[{"name":"jquery\/jquery-2.1.4.min.js","fire":0},{"name":"sprintf.js","fire":1},{"name":"ajax.js","fire":0},{"name":"keyhandler.js","fire":1},{"name":"jquery\/jquery-ui-1.11.4.min.js","fire":0},{"name":"jquery\/jquery.cookie.js","fire":0},{"name":"jquery\/jquery.mousewheel.js","fire":0},{"name":"jquery\/jquery.event.drag-2.2.js","fire":0},{"name":"jquery\/jquery-ui-timepicker-addon.js","fire":0},{"name":"jquery\/jquery.ba-hashchange-1.3.js","fire":0},{"name":"jquery\/jquery.debounce-1.0.5.js","fire":0},{"name":"menu-resizer.js","fire":1},{"name":"cross_framing_protection.js","fire":0},{"name":"rte.js","fire":1},{"name":"tracekit\/tracekit.js","fire":1},{"name":"error_report.js","fire":1},{"name":"config.js","fire":1},{"name":"doclinks.js","fire":1},{"name":"functions.js","fire":1},{"name":"navigation.js","fire":1},{"name":"indexes.js","fire":1},{"name":"common.js","fire":1},{"name":"page_settings.js","fire":1},{"name":"db_structure.js","fire":1},{"name":"tbl_change.js","fire":1}],"_selflink":"db_structure.php?db=panel&table=&server=1&target=&token=3f1b9ae121d777f7b34f410c3ee1eaca","_displayMessage":"","_debug":"'null'","_errors":"<div class=\"clearfloat\" id=\"pma_errors\"><\/div>","_promptPhpErrors":false,"_reloadQuerywindow":{"db":"panel","table":"","sql_query":""},"_params":{"common_query":"?token=3f1b9ae121d777f7b34f410c3ee1eaca","opendb_url":"db_structure.php","safari_browser":0,"collation_connection":"utf8mb4_unicode_ci","lang":"de","server":"1","table":"","db":"panel","token":"3f1b9ae121d777f7b34f410c3ee1eaca","text_dir":"ltr","show_databases_navigation_as_tree":true,"pma_text_default_tab":"Anzeigen","pma_text_left_default_tab":"Struktur","pma_text_left_default_tab2":false,"LimitChars":50,"pftext":"","confirm":true,"LoginCookieValidity":1440,"logged_in":true,"PMA_VERSION":"4.6.0-rc1","auth_type":"cookie"}}

```
### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

### Server configuration
**Operating system:Debian8**

**Web server:Apache2.2**

**Database:Mysql 5.5**

**PHP version:7**

**phpMyAdmin version:4.6.0rc1**

### Client configuration
**Browser:Chrome**

**Operating system:Windows 10**

**Comment by ibennetch** [comment]
Hi, and thanks for your report.

This appears to be a duplicate of https://github.com/phpmyadmin/phpmyadmin/issues/12055 and I expect to release -rc2 later today as a fix.

Thanks again for the detailed report!

**Comment by ibennetch** [comment]
I've just released version 4.6.0-rc2 to address this. Thanks.

**Comment by 2002demon** [comment]
Thank You, rc2 is OK.
